pkg/rescheduler/strategies/lownodeutilization.go

```diff
@@ -216,15 +224,14 @@ func NodeUtilization(node *v1.Node, pods []*v1.Pod) (api.ResourceThresholds, []*
 	bePods := []*v1.Pod{}
 	nonRemovablePods := []*v1.Pod{}
 	otherPods := []*v1.Pod{}
-
 	totalReqs := map[v1.ResourceName]resource.Quantity{}
 	for _, pod := range pods {
 		sr, err := podutil.CreatorRef(pod)
 		if err != nil {
 			sr = nil
 		}
 
-		if podutil.IsMirrorPod(pod) || podutil.IsPodWithLocalStorage(pod) || sr == nil || podutil.IsDaemonsetPod(sr) {
+		if podutil.IsMirrorPod(pod) || podutil.IsPodWithLocalStorage(pod) || sr == nil || podutil.IsDaemonsetPod(sr) || podutil.IsCriticalPod(pod) {
```

Comment:
I am still wondering why this is happening.

Reply:
I am confused too.